### PR TITLE
Deactivate UI 

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6270 
https://github.com/bcgov/entity/issues/5937

*Description of changes:*

Deactivate UI changes

More tests are coming soon

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue [hunk 1]
```diff
@@ -141,8 +141,8 @@
       <!-- Leave Account -->
       <v-btn
         icon
-        :aria-label="canDissolve() ? 'Dissolve Account' : 'Leave Account'"
-        :title="canDissolve() ? 'Dissolve Account' : 'Leave Account'"
+        :aria-label="'Leave Account'"
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue:144]: no need of :area-label instead can we write like this aria-label="Leave Account"

File: auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue [hunk 2]
```diff
@@ -141,8 +141,8 @@
...
+        :title="'Leave Account'"
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue:145]: same like above

File: auth-web/src/util/common-util.ts [hunk 3]
```diff
@@ -95,6 +95,10 @@ export default class CommonUtils {
   static formatDatePickerDate (date?: Date) {
     return moment(date || new Date()).format('YYYY-MM-DD')
   }
+  // Formatting date in the desired format for vue date pickers
+  static formatCurrentDate () {
+    return moment(new Date()).format('MM/DD/YYYYs')
```
Comment (shabeeb-aot) [inline auth-web/src/util/common-util.ts:100]: last 's' MM/DD/YYYYs intended?

File: auth-web/src/views/auth/AccountDeactivate.vue [hunk 4]
```diff
@@ -0,0 +1,247 @@
+<template>
+  <v-container>
+    <nav
+      class="crumbs py-6"
+      aria-label="breadcrumb">
+      <div>
+        <router-link :to=accountInfoUrl>
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/AccountDeactivate.vue:7]: :to="accountInfoUrl"

File: auth-web/src/views/auth/AccountDeactivate.vue [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+      title="You can't currently deactivate account"
+      dialog-class="notify-dialog"
+      max-width="640"
+      :text="message"
+    >
+      <template v-slot:icon>
+        <v-icon large color="error">mdi-information-outline</v-icon>
+      </template>
+      <template v-slot:actions>
+        <v-btn large color="primary" data-test="deactivate-btn" class="mr-5" :to=accountInfoUrl>
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/AccountDeactivate.vue:120]: :to="accountInfoUrl"

File: auth-web/src/views/auth/AccountDeactivate.vue [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+    {
+      text: 'deactivatePadConfirmation',
+      type: 'PREMIUM',
+      selected: false
+    }
+
+  ]
+
+  private async confirm () {
+    // eslint-disable-next-line no-console
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/AccountDeactivate.vue:178]: remove un-used

**Review by shabeeb-aot — APPROVED** [review]

File: auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue:144]: Yup..good catch ..will change it

**Review by karthik-aot — APPROVED** [review]

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**